src/ValueObject/ImageValueObject.php

```diff
@@ -144,4 +144,25 @@ public static function fromImageItem(ImageItem $image_item): ValueObjectInterfac
     );
   }
 
+  /**
+   * Construct object from a styled image.
```

Comment:
Here you are still constructing from a Drupal image field and pass it through an image style.
I would change the comment according to this.